tests/Spec.elm

```diff
@@ -69,6 +95,26 @@ suite =
           in Expect.equal
             (get (r.foo << try << r.bar) updatedExample)
             Nothing
+      , describe "dict"
+            [ test "set currently present to present" <| \_ ->
+                  let updatedDict = set (dictEntry "foo") (Just 9) dict
```

Comment:
I had some trouble getting the formatting exactly the same as it was.

If you want, we could do a project-wide elm-format before merging this in.

Reply:
It's ok as it is, it compiles which is great, and I can read it which is good too :)

You're not the first person to suggest elm-format.
I don't have it installed currently, so it's a bit hard for me to do, but if it helps people use and contribute to the library, that seems like a plus.